src/jit/arraystack.h

```diff
@@ -141,6 +121,13 @@ class ArrayStack
         return data[indx];
     }
 
+    // return a reference to the i'th from the bottom
+    T BottomRef(int indx)
```

Comment:
@sandreenko This should have been `T&` :grin:. Unfortunately I only noticed this now, when you merged and I saw a conflict with my own `BottomRef` in another PR.

Reply:
Thank you. I will use this function in my next PR and fix it there.